includes/Classifai/Providers/Azure/TextToSpeech.php

```diff
@@ -0,0 +1,553 @@
+<?php
+/**
+ * IBM Watson NLU
+ */
+
+namespace Classifai\Providers\Azure;
+
+use Classifai\Providers\Provider;
+use Classifai\Blocks;
+use WP_Http;
+
+use function Classifai\get_post_types_for_language_settings;
+
+class TextToSpeech extends Provider {
+
+	const API_PATH = 'cognitiveservices/v1';
+
+	const SYNTHESIZE_SPEECH_KEY = '_classifai_synthesize_speech';
+
+	const AUDIO_ID_KEY = '_classifai_post_audio_id';
+
+	const AUDIO_TIMESTAMP_KEY = '_classifai_post_audio_timestamp';
+
+	/**
+	 * Watson NLU constructor.
+	 *
+	 * @param string $service The service this class belongs to.
+	 */
+	public function __construct( $service ) {
+		parent::__construct(
+			'Microsoft Azure',
+			'Text to Speech',
+			'azure_text_to_speech',
+			$service
+		);
+	}
+
+	/**
+	 * Enqueue the editor scripts.
+	 */
+	public function enqueue_editor_assets() {
+		global $post;
```

Comment:
We should probably add a check here to ensure this feature has been enabled for this post type before loading the JS files, so we don't end up having files loaded unnecessarily.